livekit-server/templates/service.yaml

```diff
@@ -4,12 +4,14 @@ metadata:
   name: {{ include "livekit-server.fullname" . }}
   labels:
     {{- include "livekit-server.labels" . | nindent 4 }}
-  {{- if eq .Values.loadBalancer.type "aws" }}
   annotations:
+  {{- with .Values.service.annotations}}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- if eq .Values.loadBalancer.type "aws" }}
     service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
     service.beta.kubernetes.io/aws-load-balancer-type: nlb
   {{- else if or (eq .Values.loadBalancer.type "gke") (eq .Values.loadBalancer.type "gke-managed-cert") }}
-  annotations:
     cloud.google.com/backend-config: '{"ports": {"{{ .Values.loadBalancer.servicePort }}":"{{ include "livekit-server.fullname" . }}"}}'
```

Comment:
We also need the BackendConfig in the VPC-native cluster. Maybe we need to add `loadBalancer.type=gke-vpc-native` after all?